### DISCUSSION TITLE
Optional Additional Electrical / Fire Hazards

### DISCUSSION THREAD

**Discussion by Scuba-Recon** [FakeFishGames/Barotrauma#10155] [Ideas and suggestions]
Junction boxes and Reactors seem to be the only source for potential fire hazards which seems odd that the other power fed devices couldn't have the same thing. The fire probability value should be optional for a few other machines and electrical types such as engines battery and super capacitors.

Another would be some form of fire hazard caused by fabrication where flammable fumes from it's operation on building certain items could leak into the air vent system which would potentially combust and start a fire randomly where a air vent is located on the submarine.

**Comment by Petethegoat** [comment]
Makes sense. You'd think the O2 generator and supercaps in particular would be good firesources.
